src/main/java/net/atos/zac/app/mail/MailRESTService.java

```diff
@@ -60,6 +72,16 @@ public void sendAcknowledgmentReceiptMail(@PathParam("zaakUuid") final UUID zaak
             throw new RuntimeException(String.format("email '%s' is not valid", restMailObject.ontvanger));
         }
         mailService.sendMail(restMailObject.ontvanger, restMailObject.onderwerp, restMailObject.body, restMailObject.createDocumentFromMail, zaakUuid);
+
+        final Zaak zaak = zrcClientService.readZaak(zaakUuid);
```

Comment:
Als je het ophalen van de zaak als eerste doet in deze method dan kun je de zaak, i.p.v. het zaakUuid, meegeven aan de policyService.readZaakActies() . Dat scheelt weer een call om de zaak op te halen.

Reply:
Scherp! Opgelost in nieuwe commit. Ook meteen de duplicaat call voor het ophalen van het statustype voorkomen.